src/Storage/Listener/S3Listener.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @author Florian Krämer
+ * @copyright 2012 - 2015 Florian Krämer
+ * @license MIT
+ */
+namespace Burzum\FileStorage\Storage\Listener;
+
+/**
+ * Local FileStorage Event Listener for the CakePHP FileStorage plugin
+ *
+ * @author Florian Krämer
+ * @author Tomenko Yegeny
+ * @license MIT
+ */
+class S3Listener extends LocalListener {
+
+/**
+ * Default settings
+ *
+ * @var array
+ */
+	protected $_defaultConfig = [
+		'pathBuilder' => 'S3',
+		'pathBuilderOptions' => [
+			'modelFolder' => true,
```

Comment:
Why is this true? Do you need it for your app? I don't think it should be set by default.